packages/ag-charts-website/e2e/toolbar.spec.ts

```diff
@@ -155,31 +158,44 @@ test.describe('toolbar', () => {
         await page.keyboard.press('Backspace');
         await page.keyboard.press('Home');
         await page.keyboard.press('Delete');
-        await expect(page).toHaveScreenshot('delete-erased-text.png');
+        await expect(wrapper).toHaveAttribute('data-update-pending', 'false');
+        await expect(page).toHaveScreenshot('delete-erased-text.png', { animations: 'disabled' });
```

Comment:
Is `delete-erased-text.png` the only file that is causing trouble? I'm wondering whether so many `data-update-pending` checks are necessary 🤔

Reply:
Due to the nature of the failure, it looks like some interaction overtook a hover earlier in the flow - so waiting only before the snapshot wouldn't address this.